### PR TITLE
fix #1257 and unicode error in Contrast importer

### LINKED ISSUE #1257: Contrast Scanner not available for import
**Bug description**
Contrast CSV not an available option under the Scan Type dropdown

**Steps to reproduce**
1. Go to Findings
2. Click on Import Scan Results
3. Scroll down to Scan type
4. See error - No Contrast option available.

**Expected behavior**
Contrast Scanner should be available as an import choice per the Documentation

**Environment information**

 - DefectDojo version: 1.5.4

**Screenshots** (optional)

### PR DESCRIPTION
**Pull request by dr3dd589**
fix #1257 

**Note: DefectDojo is now on Python3 and Django 2.2. Please submit your pull requests to the 'python3_dev' branch as the 'dev' branch is only for bug fixes. Any PR's submitted to the 'dev' branch should also be converted to Python3 and submited to the 'python3_dev' branch. 

When submitting a pull request, please make sure you have completed the following checklist:

- [x] Your code is flake8 compliant 
- [ ] If this is a new feature and not a bug fix, you've included the proper documentation in the ReadTheDocs documentation folder. https://github.com/DefectDojo/Documentation/tree/master/docs or provide feature documentation in the PR.
- [ ] Model changes must include the necessary migrations in the dojo/dd_migrations folder.
- [ ] Add applicable tests to the unit tests.

### CODE REVIEW

**Comment by adracea** [comment]
I'm not quite sure if this actually would still replicate on python3 ? Otherwise it looks good

**Comment by dr3dd589** [comment]
hello, @adracea I have tested it on python3 with a contrast scan sample file from the sample repo. and it's working fine.

File: dojo/tools/contrast/parser.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __init__(self, filename, test):
             impact = "N/A"
             references = "N/A"
 
-            dupe_key = hashlib.md5(category + '|' + str(cwe) + '|' + title + '|').hexdigest()
+            dupe_key = hashlib.md5(category.encode('utf-8') + str(cwe).encode('utf-8') + title.encode('utf-8')).hexdigest()
```
Comment (adracea) [inline dojo/tools/contrast/parser.py]: Quite sure this won't be needed in python 3 anymore ?
@Maffooch can you confirm ?
If so , I think we can try to sneak this into the python 2 branch ?

Comment (aaronweaver) [reply dojo/tools/contrast/parser.py]: Yes, we are still pushing patches to 2 as well.

Comment (Maffooch) [reply dojo/tools/contrast/parser.py]: @adracea These changes are very much needed in python3 branch but not in python2 branch. There was some sneaky unicode changes in python3.

**Pull request merged**